meilisearch-http/public/interface.html

```diff
@@ -147,6 +147,14 @@ <h2 class="subtitle">
   </body>
 
   <script>
+    function sanitizeHTMLEntities(str) {
+        if (str && typeof str === 'string') {
+            str = str.replace(/</g,"&lt;");
+            str = str.replace(/>/g,"&gt;");
```

Comment:
Ok nevermind, I found an ugly temporary solution to the previous highlighting problem.

```suggestion
            str = str.replace(/</g,"&lt;");
            str = str.replace(/>/g,"&gt;");
            str = str.replace(/&lt;em&gt;/g,"<em>");
            str = str.replace(/&lt;\/em&gt;/g,"<\/em>");
```

Reply:
Yes ugly :D I totally get your point on this. Covering every <> maybe is an overkill. Also again doing it on the browser is just a part of the solution. I feel like there should be a solution for the backend like we discussed before. 

You might look into Caja (framework by google) for limiting the attack surface on the web interface. They did some good job on this type of issues. Mine was really a monkey patch compared to what should be done. 

I use meilisearch to do a lot of searches on vulnerability data, and it's full of Cross Site Scripting examples and that's how I got into all of this. Sanitizing <> was enough for me, I can totally see that this won't work for everybody. :)

I'll commit the suggestion.

Reply:
Thank you for your research on MeiliSearch that is really cool!

As I write in [my previous comment](https://github.com/meilisearch/MeiliSearch/pull/539#issuecomment-603791604) I will work on fixing those security issues in the backend.

This is, as you said, a monkey [temporary] patch :)